tests/mocha/server/CollectionsTest.js

```diff
@@ -0,0 +1,32 @@
+MochaWeb.testOnly(function() {
+  describe('Collections', function() {
+
+    var Gizmos = new Mongo.Collection("gizmos");
+    Gizmos.attachSchema({
+      name: {
+        type: String,
+      },
+      createdAt: createdAtSchemaField,
+      updatedAt: updatedAtSchemaField,
+    });
+
+    it('createdAt & updatedAt autoValue functions', function() {
+      var gid = Gizmos.insert({name: "Rune"});
+      var inserted = Gizmos.findOne(gid);
+
+      chai.expect(inserted.name).to.equal("Rune");
+      chai.expect(inserted.createdAt).to.exist;
+      chai.expect(inserted.updatedAt).to.not.exist;
+
+      Gizmos.update({_id: gid}, { $set: {name: "Sven"}});
+      var updated = Gizmos.findOne(gid);
+
+      chai.expect(updated.name).to.equal("Sven");
+      chai.expect(updated.createdAt.getTime()).to.exist;
+      chai.expect(updated.updatedAt.getTime()).to.exist;
+
+      chai.expect(updated.createdAt.getTime()).to.equal(inserted.createdAt.getTime());
+      chai.expect(updated.createdAt.getTime()).to.not.be.greaterThan(inserted.createdAt.getTime());
```

Comment:
Hmmm... this line looks like it's referencing createdAt instead of updatedAt? Is that a mistake?

```javascript
chai.expect(updated.updateAt.getTime()).to.not.be.greaterThan(inserted.updateAt.getTime());
```

One thought: we could make this test asynchronous and use `setTimeout` to ensure that at least 1 millisecond passes before doing the `update`. I believe all you need to do is add a `done` callback to the test function.